template/config.json

```diff
@@ -9,5 +9,6 @@
   "googleAnalyticsId": "{{ googleAnalyticsId }}",
   "actionNetworkPetitionId": "{{ actionNetworkPetitionId }}",
   "donateUrl": "{{ donateUrl }}",
+  "mapboxToken": "pk.eyJ1Ijoia2VuZmZ0ZiIsImEiOiJjamlld2tzaTQwcXV6M3BxeXdlNXRnMXh1In0.A0ZjZjYMkyK7IvL9JLhUpg",
```

Comment:
I know we've been re-using this token everywhere, but I think we should start creating new access tokens for each site.

Reply:
Sounds good. Should this be a configurable key on site generation?